tutorials/notebooks/imx500_notebooks/pytorch/pytorch_yolov8n_pose_for_imx500.ipynb

```diff
@@ -0,0 +1,507 @@
+    "## Setup\n",
+    "### Install the relevant packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7c7fa04c9903736f",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q torch\n",
+    "!pip install onnx\n",
+    "!pip install -q pycocotools\n",
+    "!pip install 'huggingface-hub>=0.21.0'\n",
+    "!pip install --pre 'sony-custom-layers-dev>=0.2.0.dev5'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57717bc8f59a0d85",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "Install MCT (if it’s not already installed). Additionally, in order to use all the necessary utility functions for this tutorial, we also copy [MCT tutorials folder](https://github.com/sony/model_optimization/tree/main/tutorials) and add it to the system path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9728247bc20d0600",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os\n",
+    "import importlib\n",
+    "\n",
+    "if not importlib.util.find_spec('model_compression_toolkit'):\n",
+    "    !pip install model_compression_toolkit\n",
+    "!git clone https://github.com/sony/model_optimization.git temp_mct && mv temp_mct/tutorials . && \\rm -rf temp_mct\n",
+    "sys.path.insert(0,\"tutorials\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a1038b9fd98bba2",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### Download COCO evaluation set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8bea492d71b4060f",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "if not os.path.isdir('coco'):\n",
+    "    !wget -nc http://images.cocodataset.org/annotations/annotations_trainval2017.zip\n",
+    "    !unzip -q -o annotations_trainval2017.zip -d ./coco\n",
+    "    !echo Done loading annotations\n",
+    "    !wget -nc http://images.cocodataset.org/zips/val2017.zip\n",
+    "    !unzip -q -o val2017.zip -d ./coco\n",
+    "    !echo Done loading val2017 images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "084c2b8b-3175-4d46-a18a-7c4d8b6fcb38",
+   "metadata": {},
+   "source": [
+    "## Model Quantization\n",
+    "\n",
+    "### Download a Pre-Trained Model \n",
+    "\n",
+    "We begin by loading a pre-trained [YOLOv8n](https://huggingface.co/SSI-DNN/pytorch_yolov8n_640x640_bb_decoding) model. This implementation is based on [Ultralytics](https://github.com/ultralytics/ultralytics) and includes a slightly modified version of yolov8 pose-head (mainly the box decoding part) that was adapted for model quantization. For further insights into the model's implementation details, please refer to [MCT Models Garden - yolov8](https://github.com/sony/model_optimization/tree/main/tutorials/mct_model_garden/models_pytorch/yolov8).  "
```

Comment:
do you have the box decoding in the model for this version of pose? just concerning the line  "includes a slightly modified version of yolov8 pose-head (mainly the box decoding part)"